roles/confluent.variables/defaults/main.yml

```
@@ -184,6 +184,9 @@ ssl_key_password: ""
 ### Boolean stating certs and keys are already on hosts. Used with ssl_custom_certs: true.
 ssl_custom_certs_remote_src: false
 
+### Enable Hostname Aliasing for host addressing. This will enable logic, on an individual host basis, to look for the variable `hostname`, followed by the reserved variable `ansible_host` and then `inventory_hostname` to resolve the appropriate FQDN of a host to use within configuration properties.
+hostaliasing: false
```

wdyt about?
```suggestion
hostname_aliasing_enabled: false
```

Great annotation, be sure to run `python doc.py` to get the docs rewritten